vitest/utils.ts

```diff
@@ -63,11 +63,24 @@ function expectFirestoreError(error: unknown, expectedCode: string) {
   }
 }
 
+function expectFirebaseError(error: unknown, expectedCode: string) {
+  if (error instanceof FirebaseError) {
+    expect(error).toBeDefined();
+    expect(error.code).toBeDefined();
+    expect(error.code).toBe(expectedCode);
+  } else {
+    throw new Error(
+      "Expected a Firebase error, but received a different type."
```

Comment:
Probably worth logging the error here too otherwise it'll be tricky to debug in CI

Reply:
i've enhanced this with logging as well.